### DISCUSSION TITLE
No speed package

### DISCUSSION THREAD

**Discussion by dggfi** [Pycord-Development/pycord#527] [Q&A] [answered]
The README says that there is a speed option. However, when installing with pip, it throws

```
Collecting py-cord[speed]
  Downloading py_cord-1.7.3-py3-none-any.whl (786 kB)
     ...
WARNING: py-cord 1.7.3 does not provide the extra 'speed'
```

Is the speed package option implemented?

**Comment by VincentRPS** [comment]
since 2.0 is not out on pypi yet the speed version isn't.

**Comment by Dorukyum** [comment, marked as answer]
As the other answer says, speed has been added in the master branch which will be released with 2.0. You have to check the corresponding README on pypi for released versions.

**Comment by dggfi** [comment]
Ahh. Thank you
